src/ir/effects.h

```diff
@@ -66,6 +70,36 @@ struct EffectAnalyzer
   // An atomic load/store/RMW/Cmpxchg or an operator that has a defined ordering
   // wrt atomics (e.g. memory.grow)
   bool isAtomic = false;
+  bool mayThrow = false;
```

Comment:
how about just `throws` for the name here? `branches` also means "`mayBranch`" (as we set it on `br_ifs`, and we don't handle stuff like `if (0) br` which isn't reached). So the "may" is implied.

Reply:
Done